### PR TITLE
resolve CVE-2024-43485

### PR DESCRIPTION
**Pull request by eugene-doobu**
see https://github.com/advisories/GHSA-8g4q-xg66-9fp4

### CODE REVIEW

**Comment by s2quake** [comment]
I think it would be better to use 6.0.* rather than 6.0.10, like the other project files (Libplanet.csproj, Libplanet.Action.csproj, and etc).

**Comment by eugene-doobu** [comment]
@s2quake apply wildcard to package version at 00ba5bb

**Comment by s2quake** [comment]
> @s2quake apply wildcard to package version at [00ba5bb](https://github.com/planetarium/libplanet/commit/00ba5bb31db2a2384067c9431cedd3e1f7e01047)

This PR requires Changes commit.

The changes might look something like this

```markdown
Version 5.4.0
-------------

To be released.

...

### Dependencies

 -  (Libplanet.Extensions.Cocona) Changed *System.Text.Json* from [6.0.7] to
    [6.0.*].  [[#3966]]

...

[#3966]: https://github.com/planetarium/libplanet/pull/3966
```

Could you please commit the changes?

[CHANGES.md](https://github.com/planetarium/libplanet/blob/940e7a47c429f4d52325f7dce33b3c8795350b2e/CHANGES.md?plain=1#L23)

**Comment by eugene-doobu** [comment]
@s2quake thanks, I pushed commit 9b65b454d62196765c87eef0a5338dd12ec556d5

File: CHANGES.md [hunk 1]
```diff
@@ -22,8 +22,12 @@ To be released.
 
 ### Dependencies
 
+ -  (Libplanet.Extensions.Cocona) Changed *System.Text.Json* from [6.0.7] to
+    [6.0.*].  [[#3966]]
+
 ### CLI tools
 
+[#3966]: https://github.com/planetarium/libplanet/pull/3966
```
Comment (s2quake) [inline CHANGES.md:31]:
```suggestion


```

Two blank lines are required between Version 5.3.0. Please add one more line

**Comment by eugene-doobu** [comment]
@s2quake I've applied your suggestion [c7e30ee](https://github.com/planetarium/libplanet/pull/3966/commits/c7e30eec8e3cf6df543d762015f6ed0195805676)

**Review by s2quake — APPROVED** [review]

**Pull request merged**